src/test/java/com/appdirect/sdk/vendorFields/fieldsValidation/VendorFieldValidationControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package com.appdirect.sdk.vendorFields.fieldsValidation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.validateMockitoUsage;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.appdirect.sdk.vendorFields.fieldsValidation.model.VendorFieldValidation;
+import com.appdirect.sdk.vendorFields.fieldsValidation.model.VendorFieldsValidationResponse;
+import com.appdirect.sdk.vendorFields.model.FlowType;
+import com.appdirect.sdk.vendorFields.model.OperationType;
+
+@RunWith(MockitoJUnitRunner.class)
+public class VendorFieldValidationControllerTest {
+	@Mock
+	private VendorFieldValidationHandler mockVendorFieldValidationHandler;
+
+	private VendorFieldValidationController tested;
```

Comment:
Put a easy name here:
```suggestion
	private VendorFieldValidationController vendorFieldValidationController;
```

Reply:
Done